portfolio/src/main/webapp/script.js

```diff
@@ -20,7 +20,7 @@ const POST_ID = "0cb628857f3c4c77bf7f9a879a6ec21d";
 /**
  * Adds a random greeting to the page.
  */
-addRandomGreeting = () => {
+getRandomGreeting = () => {
```

Comment:
i think the fn is still called `addRandomGreeting` on the HTML page, so button clicking on that button might cause an issue if the fn does not exist anymore.

`<button onclick="addRandomGreeting()">Hello</button>` from the HTML.